### PR TITLE
docs: add electron app page

### PR DESCRIPTION
**Pull request by Resulte**
Fix: https://github.com/casdoor/casdoor-website/issues/258

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Resulte plz also add the .gif demo animation to the docs, the effect of it is very good :)

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+}
+
+ipcMain.handle("getUserInfo", async (event, clientId, clientSecret) => {
+  const code = store.get("casdoor_code");
+  const userInfo = await getUserInfo(clientId, clientSecret, code);
+  store.set("userInfo", userInfo);
+  return userInfo;
+});
+```
+Finally, you can parse the code and get the user info just followed the[ Casdoor docs](http://localhost:3000/docs/how-to-connect/oauth).
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:164]:
```suggestion
Finally, you can parse the code and get the user info just followed the[ Casdoor docs](/docs/how-to-connect/oauth).
```

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
+---
+title: Electron App
+description: An Electron app example for Casdoor
+keywords: [electron, sdk]
+---
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:5]:
```suggestion
---

```

please leave a blank line in the doc header

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+Builds the electron app and run this app.
+
+### `npm run make` or `yarn make`
+
+Package and distribute your application. It will create the `out` folder where your package will be located:
+`// Example for macOS out/  
+├── out/make/zip/darwin/x64/casdoor-electron-example-darwin-x64-1.0.0.zip  
+├── ...  
+└── out/casdoor-electron-example-darwin-x64/casdoor-electron-example.app/Contents/MacOS/casdoor-electron-example`
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:39]: image

it doesn't look good.

maybe we should use code block
````md
```
// Example for macOS out/  
├── out/make/zip/darwin/x64/casdoor-electron-example-darwin-x64-1.0.0.zip  
├── ...  
└── out/casdoor-electron-example-darwin-x64/casdoor-electron-example.app/Contents/MacOS/casdoor-electron-example
```
````

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+If you click `Login with Casdoor` botton, your default browser will be opened automatically and show the login page.
+![browser](/img/howto-desktop-electron-browser.png)
+After you login successfully, your electron application will be opened and your user name will be showed on your application.
+![electron logout](/img/howto-desktop-electron-logout.png)
+
+# How to integrate
+
+## Set the custom protocal
+
+Firstly, you need to set the custum protocal called `casdoor`.
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:54]:
```suggestion
Firstly, you need to set the custom protocol called `casdoor`.
```

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+description: An Electron app example for Casdoor
+keywords: [electron, sdk]
+---
+An [Electron app example](https://github.com/casdoor/casdoor-electron-example) for Casdoor.
+
+# How to run example
+
+## Init the example
+
+You need to init requires 6 parameters, which are all string type:
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:12]: Is this a grammatically wrong sentence?

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+Package and distribute your application. It will create the `out` folder where your package will be located:
+`// Example for macOS out/  
+├── out/make/zip/darwin/x64/casdoor-electron-example-darwin-x64-1.0.0.zip  
+├── ...  
+└── out/casdoor-electron-example-darwin-x64/casdoor-electron-example.app/Contents/MacOS/casdoor-electron-example`
+
+## Prview the example
+
+After you run this electron application, a new window will be showed on your desktop.
+![electron login](/img/howto-desktop-electron-login.png "Image not found: /img/howto-desktop-electron-login.png")
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:44]: `"Image not found: /img/howto-desktop-electron-login.png"`

what's the use of it ?

File: docs/how-to-connect/desktop-sdks/electron-app.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+If you click `Login with Casdoor` botton, your default browser will be opened automatically and show the login page.
+![browser](/img/howto-desktop-electron-browser.png)
+After you login successfully, your electron application will be opened and your user name will be showed on your application.
+![electron logout](/img/howto-desktop-electron-logout.png)
+
+# How to integrate
+
+## Set the custom protocal
```
Comment (Selflocking) [inline docs/how-to-connect/desktop-sdks/electron-app.md:52]:
```suggestion
## Set the custom protocol
```

**Comment by hsluoyz** [comment]
@Resulte plz modify based on @Selflocking 's suggestions

**Review by Selflocking — APPROVED** [review]

**Pull request merged**